### PR TITLE
Task 92t: uml of output handler

### PR DESCRIPTION
**Pull request by codeFactory12**
**Description**
Uml for the audio function of the output handler implementing basic functions for managing audio from the console.
**What did i do?**

- I make a UML for the audio implementation, its the squema for console about audio management specifing the methods will provide for his use.

**How did i do it?**

- Based on the audio and its to provide soinds by paths of the .mp3 .

**Why did i do it?**

- So the development part of the audio for the console has to be based on the UML.

### CODE REVIEW

**Review by fabianromeroclaros — APPROVED** [review]

**Review by Misax148 — APPROVED** [review]

**Comment by zrock-dev** [comment]
Following the #12 architecture update, please migrate this class diagram to "Sound Manager"

**Comment by codeFactory12** [comment]
> Following the #12 architecture update, please migrate this class diagram to "Sound Manager"

The package name changed to "Sound Manager" and the name of the directory changed to "screen-container", appliying the new architecture.

File: uml/backend-software-system/Output-audio-handler.puml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@startuml
+package "Audio Handler"{
```
Comment (zrock-dev) [inline uml/backend-software-system/Output-audio-handler.puml:2]: The package name should be "Sound Manager"  Isn't?

**Review by zrock-dev — COMMENTED** [review]
The new path should be at the same level as frontend-software-system
image

File: uml/backend-software-system/Output-audio-handler.puml (hunk 1 above)
Comment (codeFactory12) [reply uml/backend-software-system/Output-audio-handler.puml:2]: i cant update something that its already merged, so i oppened a new PR with this fixes, becasue its not charging new commits to this PR but it does to the branch related to the US

Comment (zrock-dev) [reply uml/backend-software-system/Output-audio-handler.puml:2]: oke

**Pull request merged**